spec/transformers/api/v4/user_transformer_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Api::V4::UserTransformer do
+  describe 'to_response' do
+    let!(:user) { create(:user) }
+
+    subject(:response) { described_class.to_response(user) }
+
+    before do
+      allow(Api::V4::Transformer).to receive(:to_response)
+                                       .with(user)
+                                       .and_return({ 'user' => 'attributes'})
+    end
+
+    it 'includes user attributes' do
+      expect(response).to include('user' => 'attributes')
+    end
+
+    it 'includes associated params' do
+      expect(response).to include(
+                            'registration_facility_id' => user.registration_facility.id,
+                            'phone_number' => user.phone_number,
+                            'password_digest' => user.phone_number_authentication.password_digest
+                          )
+    end
+
+    it 'excludes sensitive params' do
+      expect(response).not_to include(
+                                'otp',
+                                'otp_valid_until',
+                                'access_token',
+                                'logged_in_at',
+                                'role',
+                                'organization_id'
+                              )
```

Comment:
I noticed a problem with the spec I wrote. These attrs will of course be excluded since we're stubbing out a very simple response. Could we add these attributes to [the stub](https://github.com/simpledotorg/simple-server/pull/858/commits/7445bc8e976846b37cfca2b68b58c1fa2050324d#diff-c9b25e1050779d7ac2e974ab93d5c55eR12) so that this assertion is meaningful?

Reply:
Haha, I was just typing this out in your PR

Reply:
![great minds](https://media.giphy.com/media/jt2kGEJhTQXFhUOfRm/giphy.gif)